rbac/ledger_sync/deltas/updating.py

```diff
@@ -75,11 +78,69 @@ def get_updater(database, block_num):
     return lambda adr, rsc: _update(database, block_num, adr, rsc)
 
 
+def _update_state(database, block_num, address, resource):
+    try:
+        # update state table
+        state = dict(resource)
+        address_parts = addresser.parse(address)
+        address_binary = bytes_from_hex(address)
+        key = address_binary
+        keys = {"address": address_binary}
+        data = {
+            "block_updated": int(block_num),
+            "updated_at": r.now(),
+            "object_type": address_parts.object_type.value,
```

Comment:
Should we be using dot notation here to get the values of items in `address_parts` instead of bracket notation?